source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -242,40 +278,76 @@ void CGLTFMeshFileLoader::MeshExtractor::copyNormals(
 	
 	for (std::size_t i = 0; i < count; i++) {
 		const auto n = readVec3DF(BufferOffset(buffer,
-			3 * sizeof(float) * i));
+			3 * sizeof(float) * i), core::vector3df{1,1,1});

```

Comment:
No need for this parameter, it can call `getScale()`. If there are performance problems, we can talk about how we want to improve it.

Reply:
Fixed in e3c34cdb98785fe7747e9550d8270517de510fdb